spec/presenters/summary/html_presenter_spec.rb

```diff
@@ -99,8 +99,8 @@
     double(DateStampContext, first_name: 'Arnold', last_name: 'Slit', date_of_birth: Date.new(1990, 1, 1))
   end
 
-  let(:datastore_application) do
-    extra = {
+  let(:extra) do
```

Comment:
datastore application was a better variable name

Reply:
The variable is still there, I just extracted the extra bit into its own "let".